TRAMS-API/ODataHelpers/D365ModelHelper.cs

```diff
@@ -141,7 +150,7 @@ private List<PropertyInfo> GetTypeProperties(Type type)
         /// <returns></returns>
         private static bool IsSystemType(PropertyInfo p)
         {
-            return p.PropertyType.FullName.Contains("System.");
+            return p.PropertyType.FullName.Contains("System.") && !p.PropertyType.FullName.Contains("System.Collections.Generic.List");
```

Comment:
Question: Is there a specific reason we want to exclude Generic List? Or do we actually want to make sure we exclude all list types?  If the later perhaps we could check if the type derives some something like IEnumerable<T>.

Reply:
I'll give that a go sure